aptos-move/aptos-sdk-builder/src/golang.rs

```diff
@@ -762,18 +763,23 @@ func decode_{0}_argument(arg aptostypes.TransactionArgument) (value {1}, err err
     // - otherwise, we can use `<arg>.BcsSerialize()`, `<arg>.BcsDeserialize()` to do the work.
     fn bcs_primitive_type_name(type_tag: &TypeTag) -> Option<&'static str> {
         use TypeTag::*;
+        let str_tag: StructTag = StructTag::from_str("0x1::string::String").unwrap();
         match type_tag {
             Bool => Some("Bool"),
             U8 => Some("U8"),
             U64 => Some("U64"),
             U128 => Some("U128"),
             Address => None,
             Vector(type_tag) => match type_tag.as_ref() {
-                U8 => Some("Bytes"),
+                U8 => Some("[]uint8"),
```

Comment:
`Vec<u8>` ?

Reply:
I think the golang did some internal redefining of the types here. So here I just directly generate the targeted types in go instead of these intermediate types.